test/hummingbot/connector/exchange/ftx/test_ftx_user_stream_data_source.py

```diff
@@ -0,0 +1,320 @@
+import asyncio
+import json
+from typing import Awaitable, Optional
+from unittest import TestCase
+from unittest.mock import AsyncMock, patch
+
+from aiohttp import WSMessage, WSMsgType
+from bidict import bidict
+
+from hummingbot.client.config.client_config_map import ClientConfigMap
+from hummingbot.client.config.config_helpers import ClientConfigAdapter
+from hummingbot.connector.exchange.ftx import ftx_constants as CONSTANTS
+from hummingbot.connector.exchange.ftx.ftx_api_user_stream_data_source import FtxAPIUserStreamDataSource
+from hummingbot.connector.exchange.ftx.ftx_auth import FtxAuth
+from hummingbot.connector.exchange.ftx.ftx_exchange import FtxExchange
+from hummingbot.connector.test_support.network_mocking_assistant import NetworkMockingAssistant
+from hummingbot.core.api_throttler.async_throttler import AsyncThrottler
+
+
+class OkxUserStreamDataSourceUnitTests(TestCase):
```

Comment:
Please update the class name here

Reply:
Good catch. I will change it in the next commit.